spec/d16n-v1_0.adoc

```diff
@@ -19,6 +19,30 @@ This is written in the context of offering learning tools to children and their
 
 // TODO: write introduction
 
+It has become common to develop apps that use some form of single sign-on (SSO) to deliver functionality in institutional or educational settings.
+
+By using a single sign-on scheme, such as OpenID Connect, the third-party app often receives personal data, such as the full name, of a user during the authentication in order to present that user to other users of the app.
+As the personal data is transferred to the server of the app, any logging, caching or storage of what is received must be treated with the appropriate level of data privacy safeguards.
+When the server of the app receives this data, any logging, caching or storage it performs must be with the appropriate level of data privacy safeguards.
+
+In the education space, when building such apps for the school system this becomes problematic.
+
+// Is this true? Was it ever an alternative option for bettermarks to receive parental consent? Or is it in fact the case that we anyway get the parental consent - just for the reduced scope
+// of "anonymised" usage.
+Either each app needs to be given consent by parents to process their childrens data
```

Comment:
In the context of schools in Germany, this is not the case. The school has to get consent.
We just have to implement the data protection measures that satisfy the data controller (who we have a contract with).